packages/apps/plugins/plugin-table/src/components/ObjectTable/hooks/useObjects.ts

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { useFilteredObjects } from '@braneframe/plugin-search';
```

Comment:
was table already depending on search plugin or is this new?

Reply:
It was always there.